_components/sidenav.md

```diff
@@ -7,8 +7,6 @@ lead: Intro text on what is included in this section and how to use it. No more
 
 <div class="preview">
   
-  <h2 class="usa-heading">Sidenav</h2>
-
```

Comment:
@maya: That heading just seemed duplicative, because the page’s “Side navigation” title is just above it. But am I missing something?